src/plugins/lua/luaplugin.cpp

```diff
@@ -198,61 +258,67 @@ static bool includeTile(const Tile *tile)
     return false;
 }
 
-void LuaPlugin::writeTileset(LuaTableWriter &writer, const Tileset *tileset,
-                             unsigned firstGid)
+void LuaUtilWriter::writeTileset(LuaTableWriter &writer, const Tileset &tileset,
+                                    unsigned firstGid, bool standalone)
 {
-    writer.writeStartTable();
+    if (standalone) {
+      writer.writeStartTable();
```

Comment:
Coding style: please correct the indentation (also below).

Reply:
Done

Gah! Thought I found all of those before.